examples/CMakeLists.txt

```diff
@@ -116,7 +126,7 @@ IF (BUILD_EXAMPLES_CUDA)
                     "${FREEIMAGE_LIBRARY};${CUDA_LIBRARIES}")
             ELSE(TARGET forge)
                 BUILD_EXAMPLE(${EXAMPLE} ${FILE} ${DIR_NAME} ${FORGE_LIBRARIES}
-                    "${FREEIMAGE_LIBRARY};${GLEWmx_LIBRARY};${OPENGL_gl_LIBRARY};${X11_LIBS};${CUDA_LIBRARIES}")
+                    "${FREEIMAGE_LIBRARY};${OPENGL_gl_LIBRARY};${X11_LIBS};${CUDA_LIBRARIES}")
             ENDIF()
```

Comment:
I have a commit locally that moves
```
IF(UNIX)
             SET(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations -Wno-unused-function")
ENDIF()
```
to the outside of any build conditions from line ~130. This is currently under `BUILD_EXAMPLES_OPENCL`.

It also adds `-pthread` to the CMAKE_CXX_FLAGS. I am unable to add a comment on the specific line so adding it here.